recipes/serial/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+
+
+class ConanRecipe(ConanFile):
+    name = "serial"
+    description = "Cross-platform library for interfacing with rs-232 serial like ports"
+    topics = ("serial", "rs-232", "com")
+    homepage = "http://wjwwood.io/serial/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "Findcatkin.cmake"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
```

Comment:
Why no `shared` option? Upstream CMakeLists doesn't force STATIC.

Reply:
Correct! I will fix!

Reply:
Fixed in https://github.com/conan-io/conan-center-index/pull/4943/commits/f0869cffa133f40cded38b8bab4bf859e2c176d5